build.gradle

```diff
@@ -1,7 +1,7 @@
 buildscript {
     ext {
         jmockitVersion = "1.49"
-        springBootVersion = "3.2.5"
+        springBootVersion = "3.3.0"
```

Comment:
what is the benefit to upgrade to latest Spring Boot?
Did they change any properties that we should update?

If not, I guess we'd want to build with the minimum version of Spring Boot 3 to ensure it's gonna work for all SP 3 versions.
I don't think it really would change anything to bump this version, so I'm not strongly against, but I'm now curious to understand what you think would be the advantage.

Reply:
Main reason for update to latest Spring Boot major.minor (3.3.x as of this response) is that it keeps this library up to date in the OSS support window until May 2025; see https://spring.io/projects/spring-boot#support.